docs/software_setup/sd_card.md

```diff
@@ -1,10 +1,20 @@
 This page details the software setup required to run JetBot.
```

Comment:
Modify the introduction to the following:

# Software Setup (SD Card Image)

This page details how to set up JetBot using the pre-built JetBot SD card image.  You may prefer this option if you are new to Jetson Nano, and do not have an existing SD card configured.